spec/models/photo_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
+
+describe Photo, type: :model do
+  it { is_expected.to belong_to(:user)}
+
+  feature '#username' do
+    it 'returns the username of the owner' do
+      user = User.create(username: 'allimac',email: 'camilla@email.com', password: 'pass1234')
```

Comment:
Ups, just seen all your checks now!! Thanks, will keep in mind for the Makerthon project!